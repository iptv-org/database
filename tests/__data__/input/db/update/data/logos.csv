channel,feed,tags,width,height,format,url
002RadioTV.do,SD,,512,512,PNG,https://i.imgur.com/7oNe8xj.png
01TV.fr,SD,,512,512,PNG,https://i.imgur.com/RMucFq8.png
01TV.fr,HD,,512,512,PNG,https://i.imgur.com/aR5q6mA.png
7alaTV.ps,,,400,400,PNG,https://upload.wikimedia.org/wikipedia/commons/7/71/Nothing_whitespace_blank.png
Baladna.ps,SD,,400,400,PNG,https://upload.wikimedia.org/wikipedia/commons/7/71/Nothing_whitespace_blank.png
BeijingSatelliteTV.cn,,,512,512,PNG,https://i.imgur.com/vsktAez.png
M5.hu,SD,,512,512,PNG,https://i.imgur.com/y21wFd0.png
ZonaDAZN2.it,SD,,512,512,PNG,https://i.imgur.com/vMdyLVv.png
ZonaDAZN3.it,SD,,512,512,PNG,https://i.imgur.com/Jop3Vip.png
ZonaDAZN3.it,HD,,512,512,PNG,https://i.imgur.com/Jop3Vip.png
K04GWD1.us,,,0,0,PNG,https://upload.wikimedia.org/wikipedia/en/thumb/3/33/PBS_logo.svg/512px-PBS_logo.svg.png
K04JRD1.us,,,0,0,PNG,https://upload.wikimedia.org/wikipedia/en/thumb/3/33/PBS_logo.svg/512px-PBS_logo.svg.png
K05FWD1.us,,,0,0,PNG,https://upload.wikimedia.org/wikipedia/en/thumb/3/33/PBS_logo.svg/512px-PBS_logo.svg.png
ZugloTV.hu,,,0,0,PNG,https://i.imgur.com/wPqCtwR.png
ZutTV.ve,SD,,0,0,PNG,https://i.imgur.com/wPqCtwK.png