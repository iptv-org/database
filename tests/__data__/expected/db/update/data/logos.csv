channel,feed,tags,width,height,format,url
01TV.fr,HD,,512,512,PNG,https://i.imgur.com/aR5q6mA.png
24HorasInternacional.es,SD,light;2006;on-screen,80,80,JPEG,https://i.imgur.com/IUVRm5L.png
7alaTV.ps,HD,deprecated,80,80,JPEG,https://i.imgur.com/bw8ALGQ.jpg
Baladna.ps,HD,deprecated,80,80,JPEG,https://i.imgur.com/bw8ALGQ.jpg
beINMoviesTurk.tr,,,80,80,JPEG,https://i.imgur.com/nw8Sa2z.png
K04GWD1.us,,,80,80,JPEG,https://upload.wikimedia.org/wikipedia/commons/8/89/PBS_logo_2019.svg
K04JRD1.us,,,80,80,JPEG,https://upload.wikimedia.org/wikipedia/commons/8/89/PBS_logo_2019.svg
K05FWD1.us,,,80,80,JPEG,https://upload.wikimedia.org/wikipedia/commons/8/89/PBS_logo_2019.svg
M5.hu,HD,,512,512,PNG,https://i.imgur.com/y21wFd0.png
WenzhouEconomicandEducation.cn,,,80,80,JPEG,https://www.tvchinese.net/uploads/tv/wzjjkj.jpg
YiwuBusinessChannel.cn,,,80,80,JPEG,https://www.tvchinese.net/uploads/tv/yiwutv.jpg
YiwuNewsIntegratedChannel.cn,,,80,80,JPEG,https://www.tvchinese.net/uploads/tv/yiwutv.jpg
ZonaDAZN3.it,HD,,400,400,PNG,https://i.imgur.com/GFzANOI.png
ZTVWorld.hu,,,0,0,PNG,https://i.imgur.com/wPqCtwR.png
ZutTV.ve,HD,,0,0,PNG,https://i.imgur.com/wPqCtwK.png